config/locales/en.yml

```
@@ -147,6 +147,7 @@ en:
       find_blurb: Find, install, and publish RubyGems.
       learn:
         install_rubygems: Install RubyGems
+      mfa_banner_html: 🎉 We now support security devices! Improve your account security by <a href="/settings/edit#security-device">setting up</a> a new device.
```

We're planning to also open up a blog post? Should we have both available around the same time? If so, wdyt about a "Learn More" at the end of the message. Eg.

```suggestion
      mfa_banner_html: 🎉 We now support security devices! Improve your account security by <a href="/settings/edit#security-device">setting up</a> a new device. Learn more [here](link to blog post)!
```

---

That's a great idea. Once the blog post is up and I have a URL to link to I will update the translation with that 👍

---

Agreed +1. Let's do that.
A small adjustment - `Learn more` should be the CTA itself.

```
mfa_banner_html: 🎉 We now support security devices! Improve your account security by <a href="/settings/edit#security-device">setting up</a> a new device. [Learn more](link to blog post)!
```